### ISSUE TITLE
Overriding $sidebar-width adjusts sidebar, but not the rest of the page.

### ISSUE THREAD

**Issue by shekibobo** [activeadmin/activeadmin#3079]
Given I override `$sidebar-width` in my CSS file, the sidebar will actually overlap the main section of the body instead of resizing it appropriately.

![screen shot 2014-04-15 at 2 24 33 pm](https://cloud.githubusercontent.com/assets/293858/2711207/46ea77dc-c4cb-11e3-9412-43c619ca169c.png)

**Comment by gyuchang** [comment]
This is due to `margin-right` for `#main_content` being hard-coded as `300px` - see [app/assets/stylesheets/active_admin/structure/_main_structure.scss#L18](
https://github.com/activeadmin/activeadmin/blob/4baf474177fb6c4e2638d103dfb28c576b3084df/app/assets/stylesheets/active_admin/structure/_main_structure.scss#L18)

Change it something like below to fix the problem.
```sass
    #main_content{
      margin-right: $sidebar-width + $section-padding * 2;
    }
```

**Comment by javierjulio** [comment]
This is fixed by #3862
